src/commands/publish.ts

```diff
@@ -26,6 +26,7 @@ export const publish = async (argv: string[]) => {
   let targetItems: QiitaItem[];
   if (args["--all"]) {
     targetItems = (await fileSystemRepo.loadItems()).filter((item) => {
+      if (item.ignorePublish === false) return false;
```

Comment:
`ignorePublish`が`true`のものだけ更新対象の記事から除外したいです！

```suggestion
      if (item.ignorePublish === true) return false;
```